recipes/LibriSpeech/ASR/CTC/train_with_wav2vec.py

```diff
@@ -26,6 +26,7 @@
 from speechbrain.utils.distributed import run_on_main
 from hyperpyyaml import load_hyperpyyaml
 from pathlib import Path
+from pyctcdecode import build_ctcdecoder
```

Comment:
it should be optional nop? Now this is mandatory to pip install `pyctcdecode` in order to use the CTC wav2vec...

Reply:
Yes it should be optional, will put the import later